### PR TITLE
Chat ui bug fix

### PR DESCRIPTION
**Pull request by narender2031**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
chat-ui-bug

Chat UI bug

#FIX
Screenshot 2020-07-23 at 5 41 44 PM

Screenshot 2020-07-23 at 5 41 28 PM

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/assets/stylesheets/chat.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
   height: calc(100vh - 56px);
 
   &.live-chat--iossafari {
-    height: calc(100vh - 190px);
+    height: calc(100vh - 56px);
```
Comment (ludwiczakpawel) [inline app/assets/stylesheets/chat.scss:18]: If you ever want to reference the height of header, you could use [one of our CSS variables](https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_variables.scss#L10): `var(--header-height)`

**Comment by narender2031** [comment]
@ludwiczakpawel Can help me that how can I test this on my actual device. Are you using `ngrok`

**Comment by ludwiczakpawel** [comment]
@narender2031 I refer to @fdoxyz guide here so he can help you more than I can, but here's what I do:

1. Check your network IP: System Preferences / Network. There's an IP address in the Status section. Usually something like 192.168.0.xxx
2. Copy that IP and paste it in config/application.yml as follows:
    `APP_DOMAIN: "192.168.0.105.xip.io"`
    (yes, the trick is to use xip.io)
3. You may need to update URLs in oAuth config (whether it's Twitter or Github. for latter you can find it in [settings](https://github.com/settings/developers))
3. Run the server `bin/startup` and you should be able to enter your local env on your mobile (being on the same wifi network as your desktop) by hitting 192.168.0.xxx.xip.io:3000. 

Let me know if it works.

**Comment by citizen428** [comment]
@narender2031 & @ludwiczakpawel This has been open for a while and approved, but has a merge conflict now. Can we please try to finalize this?

**Comment by ludwiczakpawel** [comment]
As mentioned in previous comments - I think this still requires a bit work and testing that's why I requested changes while reviewing it. I'm happy to give some guidance or help with testing @narender2031 :)

**Comment by narender2031** [comment]
@ludwiczakpawel can you please help me with the testing.

**Comment by ludwiczakpawel** [comment]
@narender2031 sure thing, have you had a chance to look into what I mentioned here? https://github.com/forem/forem/pull/9466#pullrequestreview-454671898

**Comment by narender2031** [comment]
> I don't think this gonna work. Mobile layouts are very tricky and as long as it looks good on your mobile emulator on desktop, on actual device, it is much much different. They're weird things happening on actual iOS browsers (for example browser's bars disappearing while you scroll... or layout recalculation when the keyboard appears). Therefore, when I tested this PR on my actual device, 

PR is updated. I tested on my device IOS browser. The scroll issue is also fixed. Now it's working. 

image

image2

**Comment by narender2031** [comment]
There are some failing specs these are not related to the PR.

**Comment by citizen428** [comment]
I restarted the build for you @narender2031

**Comment by narender2031** [comment]
any update on this @ludwiczakpawel ?

**Review by citizen428 — APPROVED** [review]

**Review by ludwiczakpawel — APPROVED** [review]
Seems to be working well :) Cheers!

**Pull request merged**